src/secex-genprotimgvm-scripts/runvm.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
This script has really inconsistent indentation.

Reply:
Yes, i don't know what happened in the logic of my editor that it would mix tab and space indents in the same line...
Thanks for catching that.